ball/2023/index.html

```diff
@@ -76,26 +76,4 @@ <h3 class="subsubtitle question">{{ qanda.question }}</h3>
     src="//umap.openstreetmap.fr/en/map/untitled-map_736360?scaleControl=false&miniMap=false&scrollWheelZoom=false&zoomControl=true&allowEdit=false&moreControl=false&searchControl=null&tilelayersControl=null&embedControl=null&datalayersControl=null&onLoadPanel=undefined&captionBar=false&datalayers=2168470#13/52.4652/-1.9053">
     </iframe>
 </div>
-<hr>
-
-<script>
-    const countDownDate = new Date("Apr 26, 2023 19:00:00").getTime();
-
-    (function updateTimer() {
-        let now = new Date().getTime();
-        let distance = countDownDate - now;
-        let days = Math.floor(distance / (1000 * 60 * 60 * 24));
-        let hours = Math.floor((distance % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60));
-        let minutes = Math.floor((distance % (1000 * 60 * 60)) / (1000 * 60));
-        let seconds = Math.floor((distance % (1000 * 60)) / 1000);
-
-        document.getElementById("count").innerHTML = days + " <span>days</span> " + hours + " <span>hours</span> "
-            + minutes + " <span>minutes</span> " + seconds + " <span>seconds</span>";
-
-        if (distance < 0) {
-            document.getElementById("count").innerHTML = "<a href='https://cssbham.com/join'>cssbham.com/join</a>";
-        } else {
-            setTimeout(updateTimer, 1000)
-        }
-    })();
-</script>
```

Comment:
This should not be removed from either year.

Reply:
why does it need to stay? it looks redundant to me since the date is in the past

Reply:
The script includes a check that if the date is passed the countdown is hidden. Removing it isn't neccessary and isn't something we've done in previous years

Reply:
I don't see why we need to keep it but I have readded it now, but changed it so it shows the attendee count instead of the join link